src/DssSpell.sol

```diff
@@ -57,105 +44,192 @@ contract DssSpellAction is DssAction {
     //
     uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
 
-    // Math
-    uint256 constant MILLION = 10**6;
-    uint256 constant WAD     = 10**18;
-    uint256 constant RAD     = 10**45;
-
     // Addresses
-    address constant RWA002_OPERATOR         = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
-    address constant RWA002_GEM              = 0xAAA760c2027817169D7C8DB0DC61A2fb4c19AC23;
-    address constant MCD_JOIN_RWA002_A       = 0xe72C7e90bc26c11d45dBeE736F0acf57fC5B7152;
-    address constant RWA002_A_URN            = 0x225B3da5BE762Ee52B182157E67BeA0b31968163;
-    address constant RWA002_A_INPUT_CONDUIT  = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
-    address constant RWA002_A_OUTPUT_CONDUIT = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
+    address constant MCD_DOG              = 0x135954d155898D42C90D2a57824C690e0c7BEf1B;
+    address constant MCD_END              = 0xFB4713C20CCD81c53BC55e12923d37a4f0DB3E93;
+    address constant MCD_ESM              = 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267;
```

Comment:
:heavy_check_mark: no wards
```
0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267
```

Reply:
Confirming parameters:

```
seth call 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267 "min()" | seth --to-dec | seth --to-fix 18
75000.000000000000000000

seth call 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267 "proxy()"
0x000000000000000000000000be8e3e3618f7474f8cb1d074a26affef007e98fb

seth call 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267 "end()"
0x000000000000000000000000cc669f5a73e18515059f6fa869fb559a823a0cbb

seth call 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267 "gem()"
0x0000000000000000000000009f8f72aa9304c8b593d555f12ef6589cc3a579a2
```

The `end()` return is wrong, it is actually the `EndFab`--we need to redeploy.

Reply:
Never mind my comment, was somehow on an old version where Emilio's wasn't showing up.

Reply:
Came here to say this.
```
> seth call 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267 "end()(address)"
0xCc669F5A73E18515059F6fa869fb559A823A0cbB
```

Reply:
Good catch, I'll set a new ESM

Reply:
This was done.